reference/tools/user-guide.md

```diff
@@ -0,0 +1,187 @@
+---
+title: TiDB Ecosystem Tools User Guide
+category: reference
+aliases: ['/docs-cn/how-to/migrate/from-mysql/', '/docs-cn/how-to/migrate/incrementally-from-mysql/', '/docs-cn/how-to/migrate/overview/', '/docs-cn/reference/tools/use-guide/']
+---
+
+# TiDB Ecosystem Tools User Guide
+
+Currently TiDB contains multiple ecosystem tools. Some of these have overlapping functionality, some are different versions of same tools. This document gives introductions of each tools, illustrates the relationships between the tools, and describes the applicable tools specified for each scenarios and each TiDB versions.
+
+## TiDB ecosystem tools overview
+
+TiDB ecosystem tools can be divided into:
+
+- Data import tools, including full import tools, backup and restore tools, incremental import tools, and so forth.
+- Data export tools, including full export tools. incremental export tools, and so forth.
+
+The two types of tools are discussed in some details below.
+
+### Data import tools
+
+#### Full import tool loader (Stop maintenance, not recommended)
```

Comment:
If Loader is deprecated, it is better to not lead with it, but instead lead with lightning and then mention that loader still exists but is deprecated.

Reply:
We could consider even having a separate section for deprecated tools.

Reply:
Thanks a lot.